Gruntfile.js

```diff
@@ -41,12 +40,12 @@ module.exports = function (grunt) {
 
     lintThese: [
       'Gruntfile.js',
-      '<%= root %>/tasks/**/*.js',
-      '<%= root %>/test/**/*.js',
-      '<%= src %>/**/*.js',
-      '!<%= src %>/ui/public/angular-bootstrap/**/*.js',
-      '!<%= src %>/fixtures/**/*.js',
-      '!<%= root %>/test/fixtures/scenarios/**/*.js'
+      'bin',
+      'config',
+      'src',
+      'tasks',
+      'test',
+      'utilities',
```

Comment:
We can use `**` instead of the `root` reference, too:

```
    lintThese: [
      '**/Gruntfile.js',
      '**/bin/**/*.js',
      '**/config/**/*.js',
      '**/src/**/*.js',
      '**/tasks/**/*.js',
      '**/test/**/*.js',
      '**/utilities/**/*.js',
    ],
```